tests/runner/src/tests/concurrent.rs

```diff
@@ -0,0 +1,81 @@
+#![allow(dead_code)]
+use fluvio::{Fluvio, Offset};
+use futures_lite::StreamExt;
+use fluvio_future::task::spawn;
+use md5::Digest;
+use std::sync::mpsc::{Receiver, Sender};
+use fluvio_test_util::test_meta::TestOption;
+
+const TOPIC: &str = "test-bug";
+const PARTITION: i32 = 0;
+
+type Record = Vec<u8>;
+
+use fluvio_integration_derive::fluvio_test;
+
+#[fluvio_test]
```

Comment:
In this case, topic requirement should be specified in procedure macro and test runner should set that up.

Reply:
Each test should be able to specify what topic they needed.  Before test is invoked, all specifies should be provisioned, so tests are ensure that topics are ready to be produce and consume